interface/code_systems/cqm_valueset_howto.php

```diff
@@ -23,19 +23,23 @@
 
 
 require_once("../../interface/globals.php");
+use OpenEMR\Core\Header;
```

Comment:
Just for clarity in coding, generally have a blank line between the require_once section and the use section.